### DISCUSSION TITLE
¿Cómo implemento Vuetify en Vue 3 y Laravel 9?

### DISCUSSION THREAD

**Discussion by MadBrain4** [community/community#27559] [Codespaces]
Soy nuevo con Vuetify y Vue, y estoy tratando de implementarlo en una pagina con Laravel 9.

¿Podrían decirme como lo podría hacer?

Ya tengo instalado el Vuetify con

Vue add Vuetify

Este es mi app.js y la organización de las carpetas

![vue (1)](https://user-images.githubusercontent.com/96513678/181684327-8742e418-2ade-49d1-bc60-926f0dc7c494.jpg)

En src, esta la carpeta de assets, components y plugins de Vuetify

**Comment by jfullstackdev** [comment]
maybe this routing will help 

https://github.com/tighten/ziggy
